addons/website/views/snippets/s_card_offset.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_card_offset" name="Card Offset">
+    <section class="s_card_offset o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="9">
+                <div class="o_grid_item o_grid_item_image g-height-8 g-col-lg-12 col-lg-12 rounded d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 1; grid-area: 2 / 1 / 10 / 13; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px; border-radius: 6.4px !important;">
```

Comment:
If you define a `rounded` class, I don't think you have to redefine the border-radius in inline css, to be tested.
```suggestion
                <div class="o_grid_item o_grid_item_image g-height-8 g-col-lg-12 col-lg-12 rounded d-lg-block d-none o_snippet_mobile_invisible" style="z-index: 1; grid-area: 2 / 1 / 10 / 13; --grid-item-padding-x: 0px; --grid-item-padding-y: 0px;">
```